migrations/v4.js

```diff
@@ -0,0 +1,82 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin, getCourse, getComponents, testStopWhere, testSuccessWhere } from 'adapt-migrations';
+import _ from 'lodash';
+
+describe('adapt-contrib-accordion - v2.1.0 > v4.0.0', async () => {
+  let course, courseAccordionGlobals, accordions;
+  const oldAriaRegion = 'This component requires you to answer the question by selecting the relevant value. After selecting a value select the submit button below.';
+  const newAriaRegion = 'Accordion. Select each button to expand the content.';
+
+  whereFromPlugin('adapt-contrib-accordion - from v2.1.0', { name: 'adapt-contrib-accordion', version: '<4.0.0' });
+
+  whereContent('adapt-contrib-accordion - where accordion', async content => {
+    accordions = getComponents('accordion');
+    return accordions.length;
+  });
+
+  mutateContent('adapt-contrib-accordion - modify globals ariaRegion attribute', async (content) => {
+    course = getCourse();
+    if (!_.has(course, '_globals._components._accordion.ariaRegion')) _.set(course, '_globals._components._accordion.ariaRegion', { ariaRegion: newAriaRegion });
```

Comment:
```suggestion
    if (!_.has(course, '_globals._components._accordion.ariaRegion')) {
      _.set(course, '_globals._components._accordion.ariaRegion', newAriaRegion);
    }
```

Shouldn't be adding `ariaRegion.ariaRegion = newValue`

![image](https://github.com/user-attachments/assets/895b7289-04b6-4638-917a-9031ad1f89c1)